### PR TITLE
types: add missing `loadingScreen` type definition 

### PR DESCRIPTION
**Pull request by gekkedev**
## Types of changes

- [x] Bug fix (a non-breaking change which fixes an issue)
- [ ] New feature (a non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Description

This PR adds the missing type definition for `loadingScreen` as mentioned in #6272. The feature was implemented but that made type definition incompatible (if you decide to overwrite this setting).

## Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly. (PR: #)
  - https://github.com/nuxt/nuxtjs.org/pull/1153
- [ ] I have added tests to cover my changes (if not applicable, please state why)
- [ ] All new and existing tests are passing.

### CODE REVIEW

File: packages/types/config/build.d.ts [hunk 1]
```diff
@@ -157,6 +157,7 @@ export interface NuxtOptionsBuild {
   html?: { minify: HtmlMinifierOptions }
   indicator?: boolean
   loaders?: NuxtOptionsLoaders
+  loadingScreen?: false | {}
```
Comment (pi0) [inline packages/types/config/build.d.ts:160]:
```suggestion
  loadingScreen?: boolean| any
```

(more precisely https://github.com/nuxt/screens/blob/7495e39d27c42696b5436088a3904fd8e3871431/packages/loading/lib/module.js#L12)

Comment (gekkedev) [reply packages/types/config/build.d.ts:160]: sure, I wasn't sure anyway if these were the only allowed values :+1:

**Review by pi0 — APPROVED** [review]

**Comment by pi0** [comment]
Thanks!

**Comment by gekkedev** [comment]
Welcome; glad I could contribute something!

**Pull request merged**